velox/functions/prestosql/tests/CastBaseTest.h

```diff
@@ -35,19 +35,23 @@ class CastBaseTest : public FunctionBaseTest {
         std::make_unique<::facebook::velox::test::TestingDictionaryFunction>());
   }
 
+  core::TypedExprPtr buildCastExpr(
+      const TypePtr& fromType,
+      const TypePtr& toType,
+      bool tryCast = false) {
```

Comment:
Looks like this parameter is always passed, hence, no need to provide default value.